Task1.1/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿public class Program
+{
+    public static void Main(string[] args)
+    {
+        Console.Write("Input first number: ");
+        int a = int.Parse(Console.ReadLine());
+
+        Console.Write("Input second number: ");
+        int b = int.Parse(Console.ReadLine());
```

Comment:
a can be greater then b, or negative )) Please fix

Reply:
fixed